internal/notifier/notifier.go

```diff
@@ -442,6 +442,8 @@ func (n *Notifier) Listen(ctx context.Context, topic NotificationTopic, notifyFu
 	}
 	n.subscriptions[topic] = append(existingSubs, sub)
 
+	n.Logger.InfoContext(ctx, n.Name+": Added subscription", "new_num_subscriptions", len(n.subscriptions[topic]), "topic", topic)
```

Comment:
Found having a little more logging on subscriptions pretty handy for test debugging purposes.

Reply:
This one also feels like it should maybe be a `Debug` level given it's indicative of healthy operation and pretty low level, thoughts?

Reply:
K, made those two debug too.